cpp/daal/src/algorithms/k_nearest_neighbors/oneapi/bf_knn_classification_predict_kernel_ucapi_impl.i

```diff
@@ -88,16 +91,26 @@ Status KNNClassificationPredictKernelUCAPI<algorithmFpType>::compute(const Numer
     const Parameter * const parameter = static_cast<const Parameter *>(par);
     const uint32_t k                  = parameter->k;
 
-    const size_t nQueryRows  = ntData->getNumberOfRows();
-    const size_t nLabelRows  = labels->getNumberOfRows();
-    const size_t nDataRows   = points->getNumberOfRows() < nLabelRows ? points->getNumberOfRows() : nLabelRows;
-    const uint32_t nFeatures = points->getNumberOfColumns();
+    if (ntData->getNumberOfRows() > maxInt32SizeT || labels->getNumberOfRows() > maxInt32SizeT)
```

Comment:
Can this and the following checks be replaced by the `DAAL_CHECK` for consistency?